sorbet/rbi/shims/opentelemetry-sdk.rbi

```diff
@@ -0,0 +1,7 @@
+# typed: strong
+# frozen_string_literal: true
+
+module OpenTelemetry
+  module SDK
+  end
```

Comment:
NOTE: this can't be stubbed better, because Sorbet needs the block to be named. See:
- https://sorbet.org/docs/procs#annotating-methods-that-use-yield
- https://github.com/open-telemetry/opentelemetry-ruby/blob/afc7bc56b39adae925ebd465948a722bd64e1d2e/sdk/lib/opentelemetry/sdk.rb#L63-L66